ci-automation/capi_image.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# >>> This file is supposed to be SOURCED from the repository ROOT. <<<
+#
+# --
+function capi_image_build() {
+    # Run a subshell, so the traps, environment changes and global
+    # variables are not spilled into the caller.
+    (
+        set -euo pipefail
+
+        _capi_image_build_impl "${@}"
+    )
+}
+
+function capi_image_publish() {
+    # Run a subshell, so the traps, environment changes and global
+    # variables are not spilled into the caller.
+    (
+        set -euo pipefail
+
+        _capi_image_publish_impl "${@}"
+    )
+}
+
+function setup_capi_params() {
+    source sdk_lib/sdk_container_common.sh
+    source ci-automation/ci_automation_common.sh
+    source ci-automation/gpg_setup.sh
+    source sdk_container/.repo/manifests/version.txt
+
+    echo "==================================================================="
+    azure_profile_config_file=""
+    secret_to_file azure_profile_config_file "${AZURE_PROFILE}"
+    azure_auth_config_file=""
+    secret_to_file azure_auth_config_file "${AZURE_AUTH_CREDENTIALS}"
+
+    FLATCAR_CHANNEL="$(get_git_channel)"
+    FLATCAR_AZURE_AUTH_CREDENTIALS="${AZURE_AUTH_CREDENTIALS}"
+
+    # Provide a python3 command for the k8s schedule parsing
+    export PATH="$PATH:$PWD/ci-automation/python-bin"
+    k8s_release_versions=$(ci-automation/get_kubernetes_releases.py)
+}
+
+function _capi_image_build_impl() {
+    local arch="$1"
```

Comment:
For the high level entry point maybe let's not require the architecture to be passed from the jenkins job where it would have to have a for loop. Either this script here can build and publish arm64 or it can't because the feature is missing.